easybuild/easyconfigs/p/PyTorch/PyTorch-check-cpp-extension.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
@Flamefire Can we add a short description about what this test script does + author or upstream reference?